common/models/templates/src/trie-model.ts

```diff
@@ -453,7 +453,24 @@ class Trie {
   lookup(prefix: string): TextWithProbability[] {
     let searchKey = this.toKey(prefix);
     let rootTraversal = this.traverseFromRoot().child(searchKey);
-    return rootTraversal ? getSortedResults(rootTraversal) : [];
+
+    if(!rootTraversal) {
+      return [];
+    }
+
+    const directEntries = rootTraversal.entries;
+    // `Set` requires Chrome 38+, which is more recent than Chrome 35.
```

Comment:
If we are moving away from ES5 (#11881), does that bump our minimum version of Chrome?

Reply:
It does.  This PR's main content was written months ago, before we were looking at directly dropping it.  #11881 isn't yet merged, either.